internal/xds/translator/ratelimit.go

```diff
@@ -112,12 +113,17 @@ func (t *Translator) buildRateLimitFilter(irListener *ir.HTTPListener) *hcmv3.Ht
 			},
 			TransportApiVersion: corev3.ApiVersion_V3,
 		},
-		EnableXRatelimitHeaders: ratelimitfilterv3.RateLimit_XRateLimitHeadersRFCVersion(xRateLimitHeadersRfcVersion),
 	}
 	if t.GlobalRateLimit.Timeout > 0 {
 		rateLimitFilterProto.Timeout = durationpb.New(t.GlobalRateLimit.Timeout)
 	}
 
+	headers := xRateLimitHeadersRfcVersion
```

Comment:
```suggestion
	headers := ratelimitfilterv3.RateLimit_DRAFT_VERSION_03
```

Reply:
I am not sure we were aliasing the value for `ratelimitfilterv3.RateLimit_DRAFT_VERSION_03` to `const xRateLimitHeadersRfcVersion` but I followed that convention introducing `xRateLimitHeadersDisabled`; are you suggesting we should also remove these lines?

https://github.com/envoyproxy/gateway/blob/d5bd9235ee255c8c9aad3f018f42a84d588cafff/internal/xds/translator/ratelimit.go#L43-L55

Reply:
Yeah, we don't need to define these constants in EG again.  This may cause inconsistency.